autoPyTorch/pipeline/components/setup/network_backbone/utils.py

```diff
@@ -115,14 +139,14 @@ def shake_drop_get_bl(
         num_blocks: int,
         is_training: bool,
         is_cuda: bool
-) -> torch.tensor:
+) -> torch.Tensor:
     pl = 1 - ((block_index + 1) / num_blocks) * (1 - min_prob_no_shake)
 
     if is_training:
         # Move to torch.randn(1) for reproducibility
-        bl = torch.tensor(1.0) if torch.randn(1) <= pl else torch.tensor(0.0)
+        bl = torch.Tensor(1.0) if torch.randn(1) <= pl else torch.Tensor(0.0)
```

Comment:
Why normal distribution?
Could you give me the reference of the implementation or the paper?
I think we need the doc-string.

Reply:
You can check the implementation and the paper from [here](https://github.com/imenurok/ShakeDrop). The exactly lines from the paper are `bl ∈ {0, 1} is a Bernoulli random variable with the probability P(bl = 1) = E[bl] = pl`. I think from this we can see that the current code is doing exactly as the paper says. 

P.S. I am not the original author of this function.

Reply:
Uum, if I understand the paper correctly, this should be `torch.rand(1)`, but not `torch.randn(1)`...
Basically, this is the definition the Bernouilli random variable as well.